### PR TITLE
add navigation component

### PR DESCRIPTION
**Pull request by sk-ip**
Please review, this is the navigation component

### CODE REVIEW

File: component/navigation/navigation.css [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+}
+
+/* Typography */
+
+:root {
+    --font-family: system-ui, -apple-system, "Segoe UI", Roboto, "Helvetica Neue",
+    Arial, "Noto Sans", "Liberation Sans", sans-serif, "Apple Color Emoji",
+    "Segoe UI Emoji", "Segoe UI Symbol", "Noto Color Emoji";
+}
+
```
Comment (Aditi0505) [inline component/navigation/navigation.css]: If this is common in all css files, then you can create a common file for it and import it here.

File: component/navigation/navigation.css [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+/* Navigation */
+.nav {
+    background-color: whitesmoke;
```
Comment (Aditi0505) [inline component/navigation/navigation.css:5]: Please use hex value for consistency.

File: component/navigation/navigation.css [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+    width: fit-content;
+}
+
+.nav-list-item {
+    padding: 10px;
+    cursor: pointer;
+}
+
+.nav-list-item:hover {
+    background-color: grey;
```
Comment (Aditi0505) [inline component/navigation/navigation.css:22]: Please use hex value for consistency.

**Review by Aditi0505 — APPROVED** [review]
Left few suggestions. 
Rest looks good.
Also, you can add description and screenshots for better visualization.

**Review by ravikumar0403 — APPROVED** [review]
thoughts align with the other reviewer, rest all looks good

File: component/navigation/navigation.css (hunk 1 above)
Comment (sk-ip) [reply component/navigation/navigation.css]: thanks will do that

**Pull request merged**